Dockerfile

```diff
@@ -46,7 +46,7 @@ RUN ln -s -f /usr/bin/python3 /usr/bin/python
 
 # Install AppInsights 
 RUN pip3 install azure-storage-blob==1.4.0
-RUN pip3 install applicationinsights
```

Comment:
It's funny that I had assumed this command without the version value would always pull down the latest. Looks like that's not the case?

Reply:
Command without version always pull latest version for the which python version it support.
I added explicit version here because we need that feature and don't want to be broken later if they change the method.